config/v2_3_experimental/types/unit.go

```diff
@@ -30,14 +30,35 @@ var (
 	ErrInvalidNetworkdExt = errors.New("invalid networkd unit extension")
 )
 
+func NewNoInstallSectionError(name string) error {
+	return noInstallSectionError{name}
+}
+
+type noInstallSectionError struct {
```

Comment:
Why the custom error type? Ignition doesn't really use custom error types anywhere, so unless there's a reason for it I think I'd prefer `NewNoInstallSectionError` to just `return fmt.Errorf(..`

Reply:
Will do